src/coreclr/src/jit/flowgraph.cpp

```diff
@@ -23540,7 +23541,7 @@ void Compiler::fgInsertInlineeBlocks(InlineInfo* pInlineInfo)
 
         if (bottomBlock != nullptr)
         {
-            bottomBlock->bbFlags |= InlineeCompiler->fgLastBB->bbFlags & BBF_SPLIT_GAINED;
+            bottomBlock->bbFlags |= pInlineInfo->retBB->bbFlags & BBF_SPLIT_GAINED;
```

Comment:
will `pInlineInfo->retBB` always be non-null if `bottomBlock != nullptr`?

Reply:
`pInlineInfo->retBB` is non-null if `pInlineInfo->retExpr` is non null and we have a `noway_assert(pInlineInfo->retExpr`)` above.